setup.py

```diff
@@ -66,7 +66,7 @@ def run_tests(self):
 # especially for end-users (non-developers) who use pip to install hangups.
 install_requires = [
     'ConfigArgParse==0.11.0',
-    'aiohttp>=1.2,<2',
+    'aiohttp>=2,<3',
```

Comment:
(re-adding this comment as it was on the old code, not newly merged)

bit late raising this, but this will cause some problems for new hangoutsbot installations since we are relying on libraries that are not aiohttp 2.0 ready. may i suggest reducing the lower bound back to `aiohttp>=1.2` but keep the `<3` limit for the mean-time, otherwise some botmins who are building from source may experience issues with dependencies downstream